python/cudf/cudf/core/dataframe.py

```diff
@@ -7011,15 +7017,24 @@ def _apply_cupy_method_axis_1(self, method, *args, **kwargs):
         if skipna is not False and method in _cupy_nan_methods_map:
             method = _cupy_nan_methods_map[method]
 
+        if len(arr) == 0 and method == "nanmedian":
+            # Workaround for a cupy limitation, cupy
+            # errors for zero dim array in nanmedian
```

Comment:
Oh weird we're producing a zero dim array here? Also is this an issue we should raise to cupy that we could also link here?

Reply:
Yes, added a bug link here.